zopefoundation/zope.app.error
# Python 3.4/5/6 and PyPy support.

- 100% test coverage
- Make sure the configurations can be loaded
- Enable coverage reporting
- Rename .txt -> .rst
- A 'browser' extra contains what's needed to load the browser
  configuration, which is now conditional.
